jcl/src/java.base/share/classes/openj9/internal/foreign/abi/InternalDowncallHandler.java

```diff
@@ -27,6 +27,8 @@
 /*[ENDIF] JAVA_SPEC_VERSION >= 22 */
 import java.util.HashMap;
 import java.util.List;
+import java.util.Deque;
```

Comment:
Please move these imports inside JAVA_SPEC_VERSION >= 22 and order them alphabetically.